app/src/main/java/ca/marshallasch/veil/MainActivity.java

```diff
@@ -112,6 +113,10 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     protected void onResume() {
         super.onResume();
+        Intent intent = getIntent();
+        if(intent.getAction().equals(VeilService.NOTIFICATION_ACTION)){
```

Comment:
This does not actually navigate to view the post does it? and what happens if it is getting resumed after being opened with the notification intent from a different screen, will it still go to the dash?